infra/asttools.py

```diff
@@ -220,8 +220,8 @@ def parse(source, filename='<unknown>'):
 def inline(tree):
     tree2 = ResolveImports().visit(copy.deepcopy(tree))
     tree3 = ResolvePatches().double_visit(tree2)
-    tree4 = ResolveJSHide().visit(tree3)
-    return ast.fix_missing_locations(tree4)
+    #tree4 = ResolveJSHide().visit(tree3)
+    return ast.fix_missing_locations(tree3)
```

Comment:
Responsibility for ignoring `js_hide` methods is moved to the caller, which does so in the modified `is_outline_hide_decorator` method

Reply:
You're right, the class isn't needed now